src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py

```diff
@@ -461,7 +461,7 @@ def test_monitor_log_analytics_workspace_data_export(self, resource_group, accou
 
         self.cmd(
             'monitor log-analytics workspace data-export create -g {rg} --workspace-name {workspace_name} -n {data_export_name} '
-            '--destination {eventhub_id} --all --enable false',
+            '--destination {eventhub_id} --enable false -t {table_name}',
```

Comment:
This modify is related to a breaking change, described in https://github.com/Azure/azure-cli/issues/15742